man/journal-iterate-poll.c

```diff
@@ -0,0 +1,23 @@
+#include <poll.h>
+#include <systemd/sd-journal.h>
+
+int wait_for_changes(sd_journal *j) {
+  struct pollfd pollfd;
```

Comment:
2ch indenting… urks!

Reply:
I tried with 8 spaces, but it just too wide for a man page.